bazaarvoice/emodb
# PD-3938 retryRequest redefinition has been removed

## Github Issue #

[PD-3938](https://github.com/bazaarvoice/emodb/issues/PD-3938)

## What Are We Doing Here?
There was redefinition of retryRequest function that is not needed. It is removed to avoid Unhandled Promise rejection. 

## How to Test and Verify

1. Check out this PR
2. Import postman collection with introduced change, Run collection
3. All tests should be passed no Unhandled Promise Rejection is in the log file

## Risk
no risks

### Level 

`Low`

### Required Testing

`Regression` to recheck that other postman tests are not broken

### Risk Summary

Tests from the same collection can be affected and have some new tests failures.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
